packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.html

```diff
@@ -0,0 +1,36 @@
+<template>
+    <template if:true={hasError}>
+        <div class="slds-grid slds-wrap">
+            <div class="slds-col slds-size_12-of-12">
+                <h3 class="slds-text-heading_medium slds-text-color_error slds-text-align_center slds-text-title_bold slds-var-m-around_small">{errorTitle}</h3>
+                <p class="slds-text-align_center">{description}</p>
+            </div>
+            <template if:true={link}>
+                <div class="slds-col slds-size_12-of-12">
+                  <a class="slds-m-vertical_x-small slds-align_absolute-center" href={link}>Coveo Online Help</a>  
+                </div>
+            </template>
+            <template if:false={link}>
+                <template if:false={showMoreInfo}>
+                    <div class="slds-m-vertical_x-large slds-align_absolute-center">
+                        <lightning-button variant="brand" label="More Information"  class="slds-m-around_x-small" onclick={onClick}></lightning-button>
+                    </div>
+                </template>
+            </template>
+            <template if:true={showMoreInfo}>
+                <div class="slds-col slds-size_12-of-12 slds-align_absolute-center slds-m-top_medium">
+                   <pre class="pre_class">
+                    <code>{error}</code>
+                </pre> 
+                </div>
+                <div class="slds-align_absolute-center slds-var-m-vertical_large"><span>Check the <a href="https://connect.coveo.com/s/discussions">community</a> for similar errors or <a href="https://connect.coveo.com/s/article/5382">contact Coveo Support team</a> for more information.</span></div>
+                <div class="slds-col slds-size_12-of-12 slds-align_absolute-center slds-var-m-vertical_x-large"><lightning-button variant="brand" label="Go back" icon-name="utility:back" class="slds-m-left_x-small" onclick={onClick}></lightning-button></div>
+            </template>
+            <template if:false={showMoreInfo}>
+                <div class=" slds-col slds-size_12-of-12 slds-var-m-vertical_x-large slds-illustration slds-illustration_small">
```

Comment:
Why is this element in its own `template` element? I would tend to put it right under the "More information" button.

Reply:
the button "More information" is visible in case where `link` is `null `,`undefined` or `empty`, it s double condition display